td-reproducible-tool/readme.md

```diff
@@ -23,6 +23,8 @@ Solution: <br>
 ```
 NOTE: 3 and 4 may be resolved directly after rust-lang RFC 3127. This tool provide a temp solution to solve 3 and 4 before RFC is implemented.
 
+NOTE: Since there is a [bug](https://github.com/confidential-containers/td-shim/issues/272) that maybe was introduced by the strip option, So now rolling back temporarily.

```

Comment:
I recommend use below:

NOTE: 1 and 2 are not applied because this strip feature has potential stability issue. For example, [bug](https://github.com/confidential-containers/td-shim/issues/272). It can be enabled after the strip becomes more robust.

Reply:
> I recommend use below:
> 
> NOTE: 1 and 2 are not applied because this strip feature has potential stability issue. For example, [bug](https://github.com/confidential-containers/td-shim/issues/272). It can be enabled after the strip becomes more robust.

Adopted, thank you for your advice!